demos/tensorflow-image-tagging/project.clj

```diff
@@ -0,0 +1,29 @@
+;;
+;;       Copyright (c) 2017 Two Sigma Investments, LP.
+;;       All Rights Reserved
+;;
+;;       THIS IS UNPUBLISHED PROPRIETARY SOURCE CODE OF
+;;       Two Sigma Investments, LP.
+;;
+;;       The copyright notice above does not evidence any
+;;       actual or intended publication of such source code.
+;;
+(defproject tensorflow-image-tagging "0.1.0-SNAPSHOT"
+  :dependencies [^{:voom {:repo "https://github.com/twosigma/jet.git" :branch "waiter-patch"}}
+                 [cc.qbits/jet "0.7.10-20170801_153701-gf517b70"]
```

Comment:
Unclear if we need our patched version.

Reply:
Hmm okay, then we will not need voom either.